db/017-add-is-active-to-locations.sql

```diff
@@ -0,0 +1,23 @@
+DO $migration$
+    DECLARE migrationId INT;
+    DECLARE lastSuccessfulMigrationId INT;
+BEGIN
+    -- The migration ID of this file
+    migrationId := 17;
+
+    -- Get the migration ID of the last file to be successfully run
+    SELECT MAX(id) INTO lastSuccessfulMigrationId
+    FROM migrations;
+
+    -- Only execute this script if its migration ID is next after the last successful migration ID
+    IF migrationId - lastSuccessfulMigrationId = 1 THEN
+        -- ADD SCRIPT HERE
+        ALTER TABLE locations 
+        ADD COLUMN is_active boolean
+        DEFAULT true;
```

Comment:
Change this to `DEFAULT false`

Reply:
Done. 

And I also added a query to set the existing Locations' is_active flag to true.